test/sslapitest.c

```diff
@@ -123,18 +141,19 @@ static int test_keylog_output(char *buffer, const SSL *ssl,
             token = strtok(NULL, " \n");
             if (!token) {
                 printf("Unexpectedly short premaster secret log.\n");
-                return -1;
+                return 0;
             }
             if (strlen(token) != 16) {
                 printf("Bad value for encrypted secret: %s\n", token);
-                return -1;
+                return 0;
             }
             token = strtok(NULL, " \n");
             if (!token) {
                 printf("Unexpectedly short premaster secret log.\n");
-                return -1;
+                return 0;
             }
             /* TODO: Can I check this sensibly? */
```

Comment:
what does this old comment refer to? maybe it can be removed now after you check the count?